test_scripts/Defects/4_5/1880_DefaultAndRPCTimeoutGENERIC_ERROR.lua

```diff
@@ -11,12 +11,15 @@ local commonDefects = require("test_scripts/Defects/4_5/commonDefects")
 local apiLoader = require("modules/api_loader")
 local api = apiLoader.init("data/MOBILE_API.xml")
 local schema = api.interface[next(api.interface)]
+local actions = require("user_modules/sequences/actions")
 
 --[[ Test Configuration ]]
 runner.testSettings.restrictions.sdlBuildOptions = { { extendedPolicy = { "PROPRIETARY", "EXTERNAL_PROPRIETARY" } } }
```

Comment:
I propose to extend coverage for HTTP mode

Reply:
Script calls specific PTU function that is not applicable for `HTTP` flow. Is is the reason why `HTTP` flow is not covered.
Update of this call will be the significant update, so propose to leave current implementation as is.